src/patternfly/components/InlineEdit/inline-edit.scss

```diff
@@ -0,0 +1,202 @@
+    --pf-c-inline-edit__input-group--item--MarginRight: 0;
+  }
+
+  .pf-c-inline-edit__action-group {
+    --pf-c-inline-edit__action-group--item--MarginRight: var(--pf-c-inline-edit__input-group--m-action-group--item--MarginRight);
+  }
+}
+
+.pf-c-inline-edit__input,
+.pf-c-inline-edit__input-group,
+.pf-c-inline-edit__action.pf-m-save,
+.pf-c-inline-edit__action.pf-m-cancel {
+  display: none;
+  visibility: hidden;
+}
+
+.pf-c-inline-edit__input,
+.pf-c-inline-edit__input-group {
+  flex: 1;
+}
+
+// Action group
+.pf-c-inline-edit__action-group {
+  display: inline-flex;
+
+  > * {
+    margin-right: var(--pf-c-inline-edit__action-group--item--MarginRight);
+  }
+
+  > :last-child {
+    --pf-c-inline-edit__action-group--item--MarginRight: 0;
+  }
+
+  &.pf-m-inline {
+    --pf-c-inline-edit__action-group--item--MarginRight: var(--pf-c-inline-edit__action-group--m-inline--item--MarginRight);
+  }
+}
+
+// Toggle group
+.pf-c-inline-edit__title-group {
+  display: inline-flex;
+
+  > * {
+    margin-right: var(--pf-c-inline-edit__title-group--item--MarginRight);
+  }
+
+  > :last-child {
+    --pf-c-inline-edit__title-group--item--MarginRight: 0;
+  }
+}
+
+.pf-m-column {
+  &.pf-c-inline-edit__value-group,
+  &.pf-c-inline-edit__input-group,
+  &.pf-c-inline-edit__action-group,
+  &.pf-c-inline-edit__title-group {
+    flex-direction: column;
+  }
+
+  &.pf-c-inline-edit__value-group {
+    --pf-c-inline-edit__value-group--item--MarginRight: 0;
+  }
+
+  &.pf-c-inline-edit__input-group {
+    --pf-c-inline-edit__input-group--item--MarginRight: 0;
+  }
+
+  &.pf-c-inline-edit__action-group {
+    --pf-c-inline-edit__action-group--item--MarginRight: 0;
+  }
+
+  &.pf-c-inline-edit__title-group {
+    --pf-c-inline-edit__title-group--item--MarginRight: 0;
+  }
+}
+
+.pf-c-inline-edit [class*="group"] {
+  align-items: baseline;
+}
+
+// Active
+.pf-c-inline-edit.pf-m-inline-edit-active,
+.pf-c-inline-edit .pf-m-inline-edit-active {
+  .pf-c-inline-edit__value,
+  .pf-c-inline-edit__value-group,
+  .pf-c-inline-edit__action-group > * {
+    display: none;
+    visibility: hidden;
+  }
+
+  .pf-c-inline-edit__input,
+  .pf-c-inline-edit__input-group,
+  .pf-c-inline-edit__action.pf-m-save,
+  .pf-c-inline-edit__action.pf-m-cancel {
+    visibility: visible;
+  }
+
+  .pf-c-inline-edit__input {
+    display: block;
+  }
+
+  .pf-c-inline-edit__input-group {
+    display: flex;
+  }
+
+  .pf-c-inline-edit__action.pf-m-save,
+  .pf-c-inline-edit__action.pf-m-cancel {
+    display: inline-block;
+  }
+
+  /* stylelint-disable selector-max-class */
+  .pf-c-inline-edit__toggle .pf-c-button.pf-m-plain {
+    color: var(--pf-c-inline-edit--m-active__toggle--Color);
+  }
+  /* stylelint-enable */
+}
+
+/* stylelint-disable selector-max-class */
+.pf-c-inline-edit__action.pf-m-save.pf-m-valid .pf-c-button.pf-m-plain {
+  --pf-c-button--m-plain--Color: var(--pf-c-inline-edit__action--m-save--m-valid--c-button--m-plain--Color);
```

Comment:
We aren't currently editing other component variables in a component. Couldn't you just apply `color` to `.pf-c-inline-edit__action.pf-m-save.pf-m-valid` and the button will inherit it?